README.md

```diff
@@ -1 +1,24 @@
 # hw01
+
+# HOMEWORK 1 BIG HEADING RAWR
+My name is Max Brown and I'm a native Chicagoan. I received by Bachelor's in Environmental Health from Colorado State University. Environmental Health actually has quite little to do with the *health of the environment*, and instead focuses on *human health from environmental influences* such as air pollution, water quality, and occupational safety. I am a second year Master's of Public Policy (MPP) in the **HARRIS SCHOOL OF PUBLIC POLICY** and our brand reps instructed us to use all-caps, bold letters when referring to the school. Unfortunately I have not yet found a way to also include the "DJ air horn" sound affect when writing the name of the school. I hope to work for local government or for a non-profit and use policy to help make the world a more equitable place. I am taking three classes this quarter:
+1. This class of course!
+2. Financial Investments for Public Policy
+3. Writing for Public Policy
+
+[Here, linked in, is a link to my LinkedIn](https://www.linkedin.com/in/maxwellmbrown94/)
+
+and here is a picture of me in a hole in the rock of the Little Grand Canyon of Southern Illinois in Shawnee National Forest this summer:
+![LGC Hole Picture](https://github.com/maxwellmbrown/hw01/blob/master/Little%20Grand%20Canyon%20Hole%20Picture.jpeg)
+
+##SECONDARY _*r e f l e c t i o n*_ SECTION
```

Comment:
I think you forgot a space after the ##. This makes it look like "##SECONDARY" on the file instead of presenting itself as a header text.

```suggestion
## SECONDARY _*r e f l e c t i o n*_ SECTION
```